helper/populate_db.py

```diff
@@ -343,29 +379,26 @@ def create_test_data(user_id: int):
         DialogOpenAnswers(users_nicedayuid=user_id, answer_value='moet voor mijn gezondheid goed', question_id=3,
                           datetime=datetime.now().astimezone(tz_nl)),
         DialogClosedAnswers(users_nicedayuid=user_id, closed_answers_id=803, datetime=datetime.now().astimezone(tz_nl)),
-        DialogClosedAnswers(users_nicedayuid=user_id, closed_answers_id=1401, datetime=datetime.now().astimezone(tz_nl)),
+        DialogClosedAnswers(users_nicedayuid=user_id, closed_answers_id=1401,
+                            datetime=datetime.now().astimezone(tz_nl)),
         UserInterventionState(users_nicedayuid=user_id, intervention_phase_id=1, intervention_component_id=5,
                               completed=False, last_time=datetime.now().astimezone(tz_nl), last_part=1),
 
-        UserPreferences(users_nicedayuid=user_id, intervention_component_id=5,
-                        recursive=True, week_days='1,2,3,4,5,6,7',
-                        preferred_time=(datetime.now().astimezone(tz_nl)+timedelta(minutes=3))),
-        UserPreferences(users_nicedayuid=user_id, intervention_component_id=7,
-                        recursive=True, week_days='1,2,3,4,5,6,7',
-                        preferred_time=(datetime.now().astimezone(tz_nl)+timedelta(minutes=4))),
-        UserPreferences(users_nicedayuid=user_id, intervention_component_id=8,
-                        recursive=True, week_days='1,2,3,4,5,6,7',
-                        preferred_time=(datetime.now().astimezone(tz_nl)+timedelta(minutes=5))),
-        UserPreferences(users_nicedayuid=user_id, intervention_component_id=9,
-                        recursive=True, week_days='1,2,3,4,5,6,7',
-                        preferred_time=(datetime.now().astimezone(tz_nl)+timedelta(minutes=6))),
-        UserPreferences(users_nicedayuid=user_id, intervention_component_id=10,
-                        recursive=True, week_days='1,2,3,4,5,6,7',
-                        preferred_time=(datetime.now().astimezone(tz_nl)+timedelta(minutes=7))),
+        UserPreferences(users_nicedayuid=user_id, intervention_component_id=13,
+                        recursive=True, week_days='1',
+                        preferred_time=(datetime.now().astimezone(tz_nl))),
+        UserPreferences(users_nicedayuid=user_id, intervention_component_id=21,
+                        recursive=True, week_days='1',
+                        preferred_time=(datetime.now().astimezone(tz_nl))),
```

Comment:
Everything on our side runs on the Amsterdam timezone. We have no way to check if the user is in a different one, because we don't access the app data. Worse case scenario, the user will receive the automatic initiation in a time which is not the pre-defined one, but the dialog can be rescheduled or ignored (in this case they are automatically rescheduled)